### ISSUE TITLE
Error while running `make all`(CentOS 7)

### ISSUE THREAD

**Issue by sandoche2k** [zonemaster/zonemaster-engine#813]
```sh
make all
cp lib/Zonemaster/Engine/Test/Address.pm blib/lib/Zonemaster/Engine/Test/Address.pm
cp lib/Zonemaster/Engine/Test.pm blib/lib/Zonemaster/Engine/Test.pm
cp lib/Zonemaster/Engine/Util.pm blib/lib/Zonemaster/Engine/Util.pm
cp lib/Zonemaster/Engine/Test/Basic.pm blib/lib/Zonemaster/Engine/Test/Basic.pm
cp lib/Zonemaster/Engine/Test/Connectivity.pm blib/lib/Zonemaster/Engine/Test/Connectivity.pm
cp lib/Zonemaster/Engine/Test/Delegation.pm blib/lib/Zonemaster/Engine/Test/Delegation.pm
cp lib/Zonemaster/Engine/Test/Nameserver.pm blib/lib/Zonemaster/Engine/Test/Nameserver.pm
cp lib/Zonemaster/Engine/Test/Zone.pm blib/lib/Zonemaster/Engine/Test/Zone.pm
cp lib/Zonemaster/Engine/DNSName.pm blib/lib/Zonemaster/Engine/DNSName.pm
cp lib/Zonemaster/Engine/Logger.pm blib/lib/Zonemaster/Engine/Logger.pm
cp lib/Zonemaster/Engine/Test/DNSSEC.pm blib/lib/Zonemaster/Engine/Test/DNSSEC.pm
cp lib/Zonemaster/Engine/Nameserver.pm blib/lib/Zonemaster/Engine/Nameserver.pm
cp lib/Zonemaster/Engine/Profile.pm blib/lib/Zonemaster/Engine/Profile.pm
cp lib/Zonemaster/Engine/ASNLookup.pm blib/lib/Zonemaster/Engine/ASNLookup.pm
cp lib/Zonemaster/Engine.pm blib/lib/Zonemaster/Engine.pm
cp lib/Zonemaster/Engine/Overview.pod blib/lib/Zonemaster/Engine/Overview.pod
cp lib/Zonemaster/Engine/Nameserver/Cache.pm blib/lib/Zonemaster/Engine/Nameserver/Cache.pm
cp lib/Zonemaster/Engine/Net/IP.pm blib/lib/Zonemaster/Engine/Net/IP.pm
cp lib/Zonemaster/Engine/NSArray.pm blib/lib/Zonemaster/Engine/NSArray.pm
cp lib/Zonemaster/Engine/Logger/Entry.pm blib/lib/Zonemaster/Engine/Logger/Entry.pm
cp lib/Zonemaster/Engine/Test/Syntax.pm blib/lib/Zonemaster/Engine/Test/Syntax.pm
cp lib/Zonemaster/Engine/Recursor.pm blib/lib/Zonemaster/Engine/Recursor.pm
cp lib/Zonemaster/Engine/Constants.pm blib/lib/Zonemaster/Engine/Constants.pm
cp lib/Zonemaster/Engine/Test/Consistency.pm blib/lib/Zonemaster/Engine/Test/Consistency.pm
cp lib/Zonemaster/Engine/Translator.pm blib/lib/Zonemaster/Engine/Translator.pm
cp lib/Zonemaster/Engine/Packet.pm blib/lib/Zonemaster/Engine/Packet.pm
cp lib/Zonemaster/Engine/Zone.pm blib/lib/Zonemaster/Engine/Zone.pm
cp lib/Zonemaster/Engine/Exception.pm blib/lib/Zonemaster/Engine/Exception.pm
cp lib/Zonemaster/Engine/TestMethods.pm blib/lib/Zonemaster/Engine/TestMethods.pm
cd share && make touch-po all
make[1]: Entering directory `/home/sandoche/zonemaster-engine/share'
Makefile:5: *** missing separator.  Stop.
make[1]: Leaving directory `/home/sandoche/zonemaster-engine/share'
make: *** [pure_all] Error 2
```

**Comment by matsduf** [comment]
When I run
```
git clean -dfx
git reset --hard
git status --ignored # Now it is clean
perl Makefile.PL
make all
```
it works as it should.

What branch and commit do you work on? This is where I am when I test:
```
$ git log -n 1
commit 81fc8abe4cbf2aa3a12d30f6d266a32c2c08ed07 (HEAD -> develop, zonemaster/develop)
Merge: 6788a7f c50213b
Author: Mats Dufberg <mats.dufberg@iis.se>
Date:   Mon Oct 19 17:59:32 2020 +0200

    Merge pull request #806 from repperille/norwegian_language_fix2
    
    Updated translations with changed parameters, texts etc.
```

**Comment by sandoche2k** [comment]
My git llog

```sh
[zonemaster-engine]$ git log -n 1
commit 81fc8abe4cbf2aa3a12d30f6d266a32c2c08ed07
Merge: 6788a7f c50213b
Author: Mats Dufberg <mats.dufberg@iis.se>
Date:   Mon Oct 19 17:59:32 2020 +0200

    Merge pull request #806 from repperille/norwegian_language_fix2

    Updated translations with changed parameters, texts etc.
```
Still have the same error, after `git clean` and `reset hard`

**Comment by matsduf** [comment]
It seems like your installation has problem with the following line in `share/Makefile`:
```
POFILES != find . -maxdepth 1 -type f -name '*.po' -exec basename {} \;
```

It is a line that we have been tweeking to make it work with different OSs. I can repeat the problem on CentOS 7. The command below works fine in `share`, but not in `Makefile`.

```
$ find . -maxdepth 1 -type f -name '*.po' -exec basename {} \;
da.po
fr.po
nb.po
sv.po
```

**Comment by matsduf** [comment]
Resolved by #814.

@sandoche2k, please check again with CentOS 7 from latest develop branch.
